pkg/resourcecreator/kafka/kafka.go

```diff
@@ -121,15 +120,20 @@ func podSpecWithKafka(ast *resource.Ast, kafkaratorSecretName string) {
 	}...)
 }
 
-func Create(source resource.Source, ast *resource.Ast, resourceOptions resource.Options, naisKafka *nais_io_v1.Kafka) error {
+func Create(source resource.Source, ast *resource.Ast, resourceOptions resource.Options, naisKafka *nais_io_v1.Kafka) {
 	if resourceOptions.KafkaratorEnabled && naisKafka != nil {
-		kafkaratorSecretName, err := generateKafkaSecretName(source.GetName(), naisKafka.Pool)
-		if err != nil {
-			return err
-		}
+		secretName := generateKafkaSecretName(source.GetName(), naisKafka.Pool)
 
-		podSpecWithKafka(ast, kafkaratorSecretName)
+		podSpecWithKafka(ast, secretName)
 		ast.Labels["kafka"] = "enabled"
+		aivenApp := aiven_nais_io_v1.NewAivenApplicationBuilder(source.GetName(), source.GetNamespace()).
+			WithSpec(aiven_nais_io_v1.AivenApplicationSpec{
+				SecretName: secretName,
+				Kafka: aiven_nais_io_v1.KafkaSpec{
+					Pool: naisKafka.Pool,
+				},
+			}).
+			Build()
```

Comment:
Er det ikke vanlig å ha punktumet på denne linjen, og ikke ovenfor? 🤔

Reply:
Nei, for da blir det syntaxfeil.

Reply:
Åja, lært noe nytt.